modules/develop/partials/run-transforms.adoc

```diff
@@ -443,7 +445,7 @@ See xref:reference:public-metrics-reference.adoc[]
 
 - Transforms have no external access to disk or network resources.
 - Only single record transforms is supported, but multiple output records from a single input record is supported. For aggregations, joins, or complex transformations, use Apache Flink.
-- Only a single output topic is supported.
+- Up to 16 output topics is supported.
```

Comment:
```suggestion
- Up to 16 output topics are supported.
```

Reply:
Actually this is 8, there is a difference than the RFC and was lowered (it's kind of an arbitrary limit)